gpkit/constraints/geometric_program.py

```diff
@@ -57,8 +57,14 @@ def __init__(self, cost, constraints, substitutions=None, verbosity=1):
         if not isinstance(constraints, ConstraintSet):
             constraints = ConstraintSet(constraints)
         list.__init__(self, [constraints])  # pylint:disable=non-parent-init-called
-        self.substitutions = substitutions if substitutions else {}
+        self.substitutions = substitutions or {}
         for key, sub in self.substitutions.items():
```

Comment:
Is this still for variable-less monomials? Let's port this helpful docstring over?

```
-                # get the value of variable-less monomials
-                # so that `x.sub({x: gpkit.units.m})`
-                # and `x.sub({x: gpkit.ureg.m})`
-                # are equivalent
```

Also, I'm quite surprised this is happening in geometric program. Smells like refactor time?